server/src/functions/upsertUserPersonalization.ts

```diff
@@ -0,0 +1,23 @@
+import { PartialModelObject as PMO } from 'objection';
+
+import { UserPersonalization } from '../models';
+import { updateUserPersonalization } from './updateUserPersonalization';
+
+export const upsertUserPersonalization = async (
+  id: string,
+  userPersonalization: PMO<UserPersonalization>,
+): Promise<UserPersonalization> => {
+  const existing = await UserPersonalization.query().findById(id);
+  // TODO: check if user.name is needed. Compare with user.ts in parent folder
```

Comment:
Remove this comment since the name attribute from user table is not applicable for user_personalization